hilti/toolchain/include/ast/location.h

```diff
@@ -44,6 +44,12 @@ class Location {
     auto from() const { return _from_line; }
     auto to() const { return _to_line; }
 
+    /**
+     * Merges this Location with the provided location. Returns new a location
+     * which encompasses the entire span.
+     */
+    Location mergedRange(const Location& loc) const;
```

Comment:
Suggest to simply call this `merge`:

```suggestion
    Location merge(const Location& loc) const;
```